k8s/logs-spack-io/deployments.yaml

```diff
@@ -0,0 +1,352 @@
+            containerPort: 80
+            protocol: TCP
+        volumeMounts:
+          - mountPath: /config/nginx.conf
+            name: config
+            subPath: nginx.conf
+          - mountPath: /var/www/html
+            name: html-files
+        env:
+        - name: MOUNT_PATH
+          value: "/data"
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: spack-aws-credentials
+              key: aws_access_key_id
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: spack-aws-credentials
+              key: aws_secret_access_key        
+      - name: matomo
+        image: matomo:3.14.1-fpm-alpine
+        env:
+        - name: MATOMO_DATABASE_HOST
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_host
+        - name: MATOMO_DATABASE_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_user
+        - name: MATOMO_DATABASE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_password
+        - name: MATOMO_DATABASE_DBNAME
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_database
+        - name: MATOMO_DATABASE_ADAPTER
+          value: mysql
+        - name: MATOMO_DATABASE_TABLES_PREFIX
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: matomo_database_tables_prefix
+        resources:
+          requests:
+            cpu: 1000m
+            memory: 8Gi
+          limits:
+            cpu: 1000m
+            memory: 8Gi
+        ports:
+        - name: http
+          containerPort: 9000
+        volumeMounts:
+        - mountPath: /usr/local/etc/php/conf.d/php-special.ini
+          name: config
+          subPath: php.ini
+        - mountPath: /var/www/html
+          name: html-files
+        - mountPath: /data
+          name: migration-logs
+      - image: mysql:8.0
+        name: mysql
+        env:
+        - name: MYSQL_ROOT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_root_password
+        - name: MYSQL_DATABASE
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_database
+        - name: MYSQL_USER
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_user
+        - name: MYSQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: logs-spack-io-secrets
+              key: mysql_password
+        ports:
+        - containerPort: 3306
+          name: mysql
+
+        # Important - this is recommended to be 500GB SSD - can we do that?
```

Comment:
Yes, though we should get together to work out some of the details.  I want to make sure you get the best storage available.